sources/packages/web/src/components/students/StudentApplicationsVersion.vue

```diff
@@ -1,47 +1,69 @@
 <template>
-  <v-table>
-    <thead>
-      <tr>
-        <th id="submitted-header">Submitted</th>
-        <th id="status-header">Status</th>
-        <th id="actions-header">Actions</th>
-      </tr>
-    </thead>
-    <tbody>
-      <tr v-if="loading">
-        <td colspan="3">
-          <v-skeleton-loader type="table-row@2" />
-        </td>
-      </tr>
-      <tr v-for="version in versions" :key="version.id">
-        <td headers="submitted-header">
-          {{ getISODateHourMinuteString(version.submittedDate) }}
-        </td>
-        <td headers="status-header">
-          {{
-            mapApplicationEditStatusForStudents(version.applicationEditStatus)
-          }}
-        </td>
-        <td headers="actions-header">
-          <v-btn
-            variant="text"
-            color="primary"
-            @click="$emit('viewApplicationVersion', version.id)"
-            >View
-            <v-tooltip activator="parent" location="start"
-              >Click to view this application version</v-tooltip
-            >
-          </v-btn>
-        </td>
-      </tr>
-    </tbody>
-  </v-table>
+  <toggle-content
+    :toggled="!filteredVersions.length && !loading"
```

Comment:
If the loading is used to display the  toggle content, then the skeleton loaders will not have any effect right?